gobblin-metrics-libs/gobblin-metrics-base/src/test/java/org/apache/gobblin/metrics/event/lineage/LineageEventTest.java

```diff
@@ -65,9 +66,9 @@ public void testEvent() {
     destination01.addMetadata(branch, "1");
     lineageInfo.putDestination(destination01, 1, state0);
 
-    Map<String, LineageEventBuilder> events = LineageInfo.load(state0);
-    verify(events.get("0"), topic, source, destination00);
-    verify(events.get("1"), topic, source, destination01);
+    Map<String, Set<LineageEventBuilder>> events = LineageInfo.load(state0);
+    verify(first(events.get("0")), topic, source, destination00);
```

Comment:
should there be new tests in here of the new putDestination logic?

Reply:
The new logic was covered by a test case in `BaseDataPublisherTest`. I thought it was a duplicate to add similar tests here again.

Reply:
ah, that's right - I tried looking but missed that. my bad.